test/TesterAzureUtils/Deployment/AzureSiloTests.cs

```diff
@@ -24,7 +25,7 @@ public class AzureSiloTests
             var config = AzureSilo.DefaultConfiguration(serviceRuntime);
             config.AddMemoryStorageProvider();
 
-            AzureSilo orleansAzureSilo = new AzureSilo(serviceRuntime);
+            AzureSilo orleansAzureSilo = new AzureSilo(serviceRuntime, TestingUtils.CreateDefaultLoggerFactory(config.CreateNodeConfigurationForSilo(nameof(AzureSiloTests)).TraceFileName));
```

Comment:
create  the loggerfactory in the constructor, and then implement IDisposable

Reply:
I switched to use output.WriteLine